README.md

```diff
@@ -1,37 +1,9 @@
 [![CodeQL](https://github.com/Azure-Samples/azure-spring-boot-samples/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/Azure-Samples/azure-spring-boot-samples/actions/workflows/codeql-analysis.yml) [![CI](https://github.com/Azure-Samples/azure-spring-boot-samples/actions/workflows/ci.yml/badge.svg)](https://github.com/Azure-Samples/azure-spring-boot-samples/actions/workflows/ci.yml) [![Markdown Links Check](https://github.com/Azure-Samples/azure-spring-boot-samples/actions/workflows/markdown-link-check.yml/badge.svg)](https://github.com/Azure-Samples/azure-spring-boot-samples/actions/workflows/markdown-link-check.yml) 
 
-# Spring Cloud Azure Samples
-- This is a sample project for demonstration purposes.   
-- The project demonstrates how to use [Spring Cloud Azure](https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html) to develop applications.    
-- The sample project contains 42 samples, and it still grows.    
-- These samples are grouped by Azure services and [Spring Cloud Azure libraries](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring).    
+# Spring Cloud Azure Samples  
+- The repository hold samples about using [Spring Cloud Azure](https://learn.microsoft.com/azure/developer/java/spring-framework/) (**SCA**) libraries. 
+- The **main** branch is using the latest stable version of **SCA**. If you want to find sample about specific version of **SCA**, please switch to corresponding tag in this repository.
 
-    > For example: [Use Spring Integration with single Azure Service Bus namespace](servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespace), it located in the path `/servicebus/spring-cloud-azure-starter-integration-servicebus`.
-    >
-    >  **servicebus**: The Azure service that the sample integrated with.
-    >
-    > **spring-cloud-azure-starter-integration-servicebus**: The dependency that the sample depends on.
+# Getting Help
+- If you have any question about using these samples, please [create an new issue](https://github.com/Azure-Samples/azure-spring-boot-samples/issues/new/choose).
 
-- The main branch is using the latest stable version of Spring Cloud Azure libraries. If you want to use another version, please checkout tag according to your required version.
-
-## Running Samples With Terraform
```

Comment:
Why delete terraform section? I think it is useful.

Reply:
Each sample's README already has content about using terraform to create required resources. It's redundant to add this information in the root README.